### PR TITLE
Proofreading Integrations and Managing your dependencies > Workflows

### PR DESCRIPTION
**Pull request by dimitrikourk**
Please have a look to make sure I have not changed the meaning anywhere. I am still learning conan myself :)

### CODE REVIEW

File: integrations/cmake.rst [hunk 1]
```diff
@@ -170,12 +170,12 @@ Let's see a more manageable library `conan's zlib package`_.
     set(ZLIB_LIBRARIES ${ZLIB_LIBRARY})
     
 
-Its not very complex, specially if you are familiar with CMake **find_path** and **find_library** functions.
+It's not very complex, especially if you are familiar with the CMake **find_path** and **find_library** functions.
 
-We are calling **find_path** pointing CMake to search in **${CONAN_INCLUDE_DIRS_ZLIB** directories, where we know that we have our headers.
-Then we are calling to **find_library** passing the library names **${CONAN_LIBS_ZLIB}** and the paths where they are **${CONAN_LIB_DIRS_ZLIB}**.
+We call **find_path**, pointing CMake to search in the **${CONAN_INCLUDE_DIRS_ZLIB** directories, where we know that we have our headers.
+Then we are call **find_library**, passing the library names, **${CONAN_LIBS_ZLIB}**, and the paths where they are located, **${CONAN_LIB_DIRS_ZLIB}**.
```
Comment (memsharded) [inline integrations/cmake.rst]: we are call => we call

File: integrations/git.rst [hunk 2]
```diff
@@ -9,19 +9,19 @@ Project layout
...
 
 
 Committed files
 ===============
 
-**Conan** generates some files than should not be committed. These file can change in different computers and should be re-generated with **conan install** command.
+**Conan** generates some files than should not be committed. These file can change in different computers and should be re-generated with the **conan install** command.
 
-We suggest committing ``conanfile.txt`` and ``conanfile.py`` and ignore the rest.
+We suggest committing ``conanfile.txt`` and ``conanfile.py`` and ignoring the rest.
```
Comment (memsharded) [inline integrations/git.rst]: and => or (not an issue of this PR)

File: manage_deps/workflows.rst [hunk 3]
```diff
@@ -39,16 +39,16 @@ So the layout will be:
...
 .. code-block:: bash
    
       $ cmake ../example-hello -G ...
       $ cmake --build . --config Release
       
-With this configuration you are typically working in just one configuration of the project, 
-that will be destroyed and create a new build with a new configuration with different settings
-if needed.
+In effect, with this setup, we have created a separate configuration of the project, without
+affecting its default one. The benefit is that we can experiment freely, and even erase it and
```
Comment (memsharded) [inline manage_deps/workflows.rst]: Not the idea, there is not a default one, so we are not creating a separate configuration. We are just building one configuration, if you want to build a different one, just remove build and temp files & restart with different conan install parameters

**Comment by memsharded** [comment]
Very good PR, lots of fixes, thanks very much! Lets merge it, I'll try to address the minor issues later.

**Comment by memsharded** [comment]
Just pushed some minor changes: https://github.com/conan-io/docs/commit/d35cdcdfb3b659763877e1972fb6469259a55e2d

File: manage_deps/workflows.rst (hunk 3 above)
Comment (dimitrikourk) [inline manage_deps/workflows.rst]: I see :) Actually this paragraph was what made me worry that I may have changed the meaning. I was a bit too creative :D

File: integrations/cmake.rst (hunk 1 above)
Comment (dimitrikourk) [inline integrations/cmake.rst]: oops! sorry :)

File: manage_deps/workflows.rst (hunk 3 above)
Comment (memsharded) [inline manage_deps/workflows.rst]: No problem, the PR has tons of good contributions, this was a very minor issue.

Comment (dimitrikourk) [inline manage_deps/workflows.rst]: thanks! Glad to help.

Comment (dimitrikourk) [inline manage_deps/workflows.rst]: By the way, for future pull requests, is it better if I make them on the "develop" branch? I've just noticed it now.

Comment (memsharded) [inline manage_deps/workflows.rst]: It depends. If it is about a feature that is being under development or not yet released, do the PR to develop. If it is about something already in production/released, to master, so users can get it asap. Thanks!

Comment (dimitrikourk) [inline manage_deps/workflows.rst]: Got it! Thanks!

**Pull request merged**